test/mlir/test/gc/Transforms/deepTileContractionNamedOp.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: gc-opt --split-input-file --deep-tile-contraction-named-op %s | FileCheck %s
+// RUN: gc-opt --split-input-file %s | FileCheck %s
```

Comment:
Missing `--deep-tile-contraction-op` here?

Reply:
Thanks for reminder!